lib/decorateWithMetrics.js

```diff
@@ -136,55 +137,83 @@ function toDuration(millis, raw = false) {
  * max, min, average, total time for each method
  * and a table of individual calls
  *
- * @param {boolean} [raw=false] setting this to true will print raw instead of human friendly times
+ * @param {Object} [options]
+ * @param {boolean} [options.raw=false] - setting this to true will print raw instead of human friendly times
  * Useful when you copy the printed table to excel and let excel do the number formatting
+ * @param {'console'|'csv'|'json'|'string'} [options.format=console] The output format of this function
+ * `string` is useful when __DEV__ is set to `false` as writing to the console is disabled, but the result of this
+ * method would still get printed as output
+ * @returns {string|undefined}
  */
-function printMetrics(raw = false) {
-    const {totalTime, summaries, lastCompleteCall = {endTime: -1}} = getMetrics();
```

Comment:
Changed this to print `Last call finished at: N/A` when there are no calls made yet (or were cleared)